webidl-util/src/main/kotlin/de/fabmax/webidl/parser/ConstantParser.kt

```diff
@@ -0,0 +1,24 @@
+package de.fabmax.webidl.parser
+
+import de.fabmax.webidl.model.IdlConstant
+
+class ConstantParser(parserState: WebIdlParser.ParserState) : ElementParser(
+    parserState,
+    WebIdlParserType.Constant
+) {
+
+    override suspend fun parse(): String {
+        popToken("const")
+        val type = parseType()
+        val tokens = popUntilPattern(";") ?: parserException("Failed parsing member")
```

Comment:
I assume the exception message should be "Failed parsing constant"

Reply:
Fixed